src/LiveComponent/src/Attribute/LivePropContext.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\LiveComponent\Attribute;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ *
+ * @experimental
+ *
+ * @internal
+ */
+final class LivePropContext
+{
+    private LiveProp $liveProp;
+    private \ReflectionProperty $reflectionProperty;
+
+    public function __construct(LiveProp $liveProp, \ReflectionProperty $reflectionProperty)
```

Comment:
As the package now requires PHP 8, wouldn't it be interesting to use [constructor property promotion](https://www.php.net/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion) here and in other classes where it can be implemented?

Reply:
Absolutely, I didn't use these PHP 8 features as it breaks the php-cs-fixer job for the mono repo. I'm not sure how we want to handle this.